executor_manager/executors/docker/executor.py

```
@@ -513,6 +508,46 @@ def _add_callback_url(self, cmd: List[str], task: Dict[str, Any]) -> None:
+        2. Continue the trace started by executor_manager
+        """
+        otel_config = get_otel_config()
+        if not otel_config.enabled:
+            return
+
+        try:
+            # Add OTEL configuration environment variables
+            # These are needed for executor to initialize OpenTelemetry
+            cmd.extend([
+                "-e", "OTEL_ENABLED=true",
+                "-e", f"OTEL_SERVICE_NAME=wegent-executor",  # Use executor-specific service name
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)